src/intseq/ops.clj

```diff
@@ -46,6 +46,20 @@
                (rest (rest stack)))
          (catch IllegalArgumentException e '(:overflow)))))
 
+(defn max- [stack]
+"Pushes the maximum of the top two items onto the top of the stack."
+  (if (< (count stack) 1)
```

Comment:
wait shouldn't the if statement be (if (< (count stack) 2) ? since max takes in two arguments?